xayagametest/testcase.py

```diff
@@ -103,9 +113,18 @@ def main (self):
     self.mainLogger = logging.getLogger ("main")
     self.mainLogger.addHandler (logHandler)
     self.mainLogger.addHandler (mainHandler)
+
+    
+    if sysconfig.get_platform() == "mingw":
+      self.mainLogger.info ("msys2 mingw windows build detected, adjustings paths") 
+      self.mainLogger.info ("setting new tempdir as %s" % tempfile.gettempdir()) 
+      self.args.xayad_binary = sysconfig.get_config_var('LIBDIR') + "/daemon/xayad"
+      self.mainLogger.info ("setting new bynary path as %s" % self.args.xayad_binary) 
+      self.mainLogger.info ("setting new game_daemon path as %s" % self.args.game_daemon)
```

Comment:
Here again, you should fix this where the argument parser default values are set.  Perhaps replace `XAYAD_BINARY_DEFAULT` with a function, and in that function check for the platform.